src/main.py

```diff
@@ -46,9 +62,38 @@ def compute_means(model: AutoModelForCausalLM, tokenizer: AutoTokenizer, text: l
 
 
 # Function to calculate the steering vector``
-def calculate_steering_vector(model: AutoModelForCausalLM, tokenizer: AutoTokenizer, text_a, text_b, batch_size=16, device='cpu') -> list[torch.Tensor]:
+def calculate_steering_vector(
+    model: AutoModelForCausalLM,
+    tokenizer: AutoTokenizer,
+    text_a,
+    text_b,
+    batch_size=16,
+    device="cpu",
+) -> list[torch.Tensor]:
     means_a = compute_means(model, tokenizer, text_a, batch_size, device)
     means_b = compute_means(model, tokenizer, text_b, batch_size, device)
 
     steering_vector = [a - b for a, b in zip(means_a, means_b)]
     return steering_vector
+
+class Output(BaseModel):
+    output: bool
+def judge_output_llm(output):
```

Comment:
- include typehints for this function.
- also, i think we want two inputs, not just one.  The question and the answer?